### PR TITLE
test: refactor SessionFilterTest by rector 0.18.6

### PR DESCRIPTION
**Pull request by datamweb**
**Description**
This PR is a refactor for testing using Rector.

See https://github.com/codeigniter4/shield/actions/runs/6641954393/job/18045603644

follow-up https://github.com/codeigniter4/shield/pull/929

**Checklist:**
- [x] Securely signed commits
- [ ] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [ ] Conforms to style guide

### CODE REVIEW

File: tests/Authentication/Filters/SessionFilterTest.php [hunk 1]
```diff
@@ -43,7 +44,7 @@ public function testFilterSuccess(): void
         $this->assertSame($user->id, auth('session')->id());
         $this->assertSame($user->id, auth('session')->user()->id);
         // Last Active should have been updated
-        $this->assertNotEmpty(auth('session')->user()->last_active);
+        $this->assertInstanceOf(Time::class, auth('session')->user()->last_active);
```
Comment (kenjis) [inline tests/Authentication/Filters/SessionFilterTest.php:47]: Did rector change this?

Comment (datamweb) [reply tests/Authentication/Filters/SessionFilterTest.php:47]: Yes, 0.18.6.

Comment (kenjis) [reply tests/Authentication/Filters/SessionFilterTest.php:47]: Okay, but in my understanding `chore:` means others like tool config change,
and it does not change the production or test code.
https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716

**Review by kenjis — APPROVED** [review]

**Review by kenjis — APPROVED** [review]

**Review by samsonasik — APPROVED** [review]

**Comment by datamweb** [comment]
@kenjis @samsonasik thanks!

**Pull request merged**